<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [ComplexTestSuite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="complextestcaseValidation(failed)">
    <classes>
      <class name="com.training.complex.tests.ELTC_Complex_SecondTC">
        <methods>
          <include name="BeforeTest"/>
          <include name="BeforeMethod"/>
          <include name="VerifyRegistration"/>
          <include name="BeforeClass"/>
          <include name="AfterMethod"/>
        </methods>
      </class> <!-- com.training.complex.tests.ELTC_Complex_SecondTC -->
      <class name="com.training.complex.tests.ELTC_Complex_FourthTC">
        <methods>
          <include name="BeforeTest"/>
          <include name="PerformTeacherTask"/>
          <include name="AfterMethod"/>
          <include name="BeforeMethod"/>
          <include name="BeforeClass"/>
        </methods>
      </class> <!-- com.training.complex.tests.ELTC_Complex_FourthTC -->
    </classes>
  </test> <!-- complextestcaseValidation(failed) -->
</suite> <!-- Failed suite [ComplexTestSuite] -->
